### ISSUE TITLE
Failing to log in will dismiss the login screen

### ISSUE THREAD

**Issue by andydotxyz** [AletheiaWareLLC/spacefynego#4]
Some times this may be desirable, but when launching the app if I get my password wrong I certainly should try again.

**Comment by stuartmscott** [comment]
Good point. I didn't want to bombard the user with the login screen, but you're right that entering an incorrect password should display an error and provide an easy way to try again.

BTW you can re-trigger the login screen by tapping any of the buttons that require a logged-in user (Add, Refresh, Providers, Account)
